src/actions/rootRequestedThunk.tsx

```diff
@@ -49,7 +46,7 @@ export const rootRequestedThunk = () => {
 
         dispatch(rootRequested());
 
-        const url: string = 'http://localhost:5000/node/' + '0' + '/children';
+        const url: string = 'http://localhost:5000/entities/' + '1' + '/children';
```

Comment:
This should be a variable later, but I guess we can use this as a starting point.